projects/core/src/occ/adapters/cart/default-occ-cart-config.ts

```diff
@@ -6,9 +6,9 @@ export const defaultOccCartConfig: OccConfig = {
       endpoints: {
         /* eslint-disable max-len */
         carts:
-          'users/${userId}/carts?fields=carts(DEFAULT,potentialProductPromotions,appliedProductPromotions,potentialOrderPromotions,appliedOrderPromotions,entries(totalPrice(formattedValue),product(images(FULL),stock(FULL)),basePrice(formattedValue,value),updateable),totalPrice(formattedValue),totalItems,totalPriceWithTax(formattedValue),totalDiscounts(value,formattedValue),subTotal(formattedValue),deliveryItemsQuantity,deliveryCost(formattedValue),totalTax(formattedValue, value),pickupItemsQuantity,net,appliedVouchers,productDiscounts(formattedValue),saveTime,user,name)',
+          'users/${userId}/carts?fields=carts(DEFAULT,potentialProductPromotions,appliedProductPromotions,potentialOrderPromotions,appliedOrderPromotions,entries(totalPrice(formattedValue),product(images(FULL),stock(FULL)),basePrice(formattedValue,value),updateable),totalPrice(formattedValue),totalItems,totalPriceWithTax(formattedValue),totalDiscounts(value,formattedValue),subTotal(formattedValue),deliveryItemsQuantity,deliveryCost(formattedValue),totalTax(formattedValue, value),pickupItemsQuantity,net,appliedVouchers,productDiscounts(formattedValue),user,saveTime,name,description)',
```

Comment:
to the review

I think it's beneficial to update and add additional properties in the default-occ-cart config as there already exist a saveTime property before, but no name and description.

Name and description was always available, but we never showed it. I believe it's best to change it here other than augmenting these to just add 2-3 additional properties.

Reply:
In the long term it would be good the have them separately, but because of time constraint I agree that we can for now keep it here